_docs/install.md

```diff
@@ -7,28 +7,33 @@ toc:
 nav: 1
 ---
 
-## Install
+Install
+-------
 
 You can install Pico either using a pre-bundled release or with composer. Pico is also available on [Packagist.org][] and may be included in other projects via `composer require picocms/pico`. Pico requires PHP 5.3+
 
-### Using a pre-bundled release
+### Using a pre-bundled release - for users
```

Comment:
Users can make their own choice, some people prefer `composer`, others pre-bundled releases and some people (even non-developers) want to use `git` (the latter is a little more complicated and requires some knowledge in `git`, but explaining this is out of Pico's scope, thus it's not part of the docs). The sections don't build on one another, they're completely independent.